languages/java/config.json

```diff
@@ -57,6 +57,12 @@
         "uuid": "2165c331-6b0c-49b8-9555-bad577ec4fc6",
         "concepts": ["strings"],
         "prerequisites": ["basics"]
+      },
+      {
+        "slug": "conditionals-ternary",
```

Comment:
The name of the exercise implies a hierarchy: conditions -> ternary. As per [the docs](https://github.com/exercism/v3/blob/master/docs/maintainers/determining-concepts.md#naming-concepts), exercise names should not use a hierarchy. I think a better name in this game would be `ternary-operators`. Note that this would also mean changing the directory name of the exercise.